README.md

```diff
@@ -257,16 +262,14 @@ A consequence of the synchronization strategy described before is that it takes
 
 This is where optimistic updates come into play. We just assume optimistically that the proposed action will be applied on the server. Therefore we can apply the action on the client directly without waiting for a `performAction` from the server.
 
-If the server rejects the proposal or a race condition occurs, the client corrects its state again.
-In our case the [optimisticActionHandler](./frontend/src/app/core/optimistic-action-handler.ts) encapsulates this functionality.
+If the server rejects the proposal or a race condition occurs, the client corrects its state again. In our case, the [optimisticActionHandler](./frontend/src/app/core/optimistic-action-handler.ts) encapsulates this functionality.
 
 The state in the frontend is not guaranteed to be correct. It is only guaranteed to automatically correct itself.
 
 If you need to read from the state to change it, you should do this inside the action reducer because the `currentState` passed into a reducer is always guaranteed to be correct.
 
 ### Performance considerations
 
--   Do _not_ save a very large JS primitve (a large string like a base64 encoded image) in a part of the state that is often modified (like the root). This primitive would be copied on each change. Instead, the primitive should be saved as part of a separate object. This makes use of the performance benefits of shallow copies.
```

Comment:
Is this no longer a problem?

Reply:
Actually, I don't know. It could be that there are (or will be) performance optimizations in V8 regarding the copying of very big primitives in the same thread.

But this is a very unique usecase and now, with the benchmarks, a performance Regression from something like this would be easy to spot. In addition it is easy to fix without big architectural changes (just wrap it in an object).